app/controllers/users_controller.rb

```diff
@@ -277,6 +268,23 @@ def sanitize_user_params
     params[:user].delete_if { |_k, v| v.blank? }
   end
 
+  def set_suggested_users
+    @suggested_users = SiteConfig.suggested_users
+  end
+
+  def default_suggested_users
+    @default_suggested_users ||= User.where(username: @suggested_users)
+  end
+
+  def determine_follow_suggestions(current_user)
+    recent_suggestions = Suggester::Users::Recent.new(
+      current_user,
+      attributes_to_select: INDEX_ATTRIBUTES_FOR_SERIALIZATION,
+    ).suggest
+
+    recent_suggestions.empty? ? default_suggested_users : recent_suggestions
```

Comment:
You could actually do presence here! I had no idea if that would work with an AR relation but it does! TIL!
```ruby
recent_suggestions.presence || default_suggested_users
```

Reply:
Nice, I should've remembered that! Gonna update with your suggestion and then merge.